src/common/containers/build-details.js

```diff
@@ -72,9 +71,12 @@ class BuildDetails extends Component {
         <BetaNotification />
         <Breadcrumbs>
           { user &&
-            <Link to={`/user/${user.login}`}>My repos</Link>
+            <BreadcrumbsLink to={`/user/${user.login}`}>My repos</BreadcrumbsLink>
           }
-          <Link to={`/user/${repository.fullName}`}>{repository.fullName}</Link>
+          {' ' /* vanilla assumes space between breadcrumb elements */}
+          <BreadcrumbsLink to={`/user/${repository.fullName}`}>{repository.fullName}</BreadcrumbsLink>
+          {' ' /* vanilla assumes space between breadcrumb elements */}
+          <BreadcrumbsLink>&nbsp;</BreadcrumbsLink> {/* hack to show arrow after last breadcrumb item */}
```

Comment:
Because we want to show an arrow after last item in the breadcrumb, we need to hardcode empty breadcrumb item.